weather-app/src/app/week-forecast/week-forecast.component.ts

```diff
@@ -1,97 +1,83 @@
-import { Component } from '@angular/core';
-import { Router } from '@angular/router';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { byCountry } from 'country-code-lookup';
+import { Observable } from 'rxjs';
 import { FavoriteCityListService } from '../services/favorite-city-list.service';
+import { ForecastService } from '../services/forecast/forecast.service';
 
 @Component({
   selector: 'app-week-forecast',
   templateUrl: './week-forecast.component.html',
   styleUrls: ['./week-forecast.component.scss']
 })
-export class WeekForecastComponent {
-  currentCity = 'Ivano-Frankivsk';
-  currentCountry = 'UA';
-  curentWeatherIconUrl = 'https://raw.githubusercontent.com/basmilius/weather-icons/029d142b34871bcbc90d7cd46081c50310c831c5/production/fill/svg/overcast-day-fog.svg';
-  columnStatus = true;
-  firstColumn = false;
-  firstColumnChecked = true;
-  secondColumn = true;
-  secondColumnChecked = false;
-  thirdColumn = true;
-  thirdColumnChecked = false;
-  fourthColumn = true;
-  fourthColumnChecked = false;
-  fifthColumn = true;
-  fifthColumnChecked = false;
-  sixthColumn = true;
-  sixthColumnChecked = false;
-  sevenColumn = true;
-  sevenColumnChecked = false;
+export class WeekForecastComponent implements OnInit {
+  public searchQuery: string = '';
+  public forecastData: Observable<Object> = new Observable();
+  public currentCity: string = '';
+  public currentCountry: string = '';
+  public forecast!: any;
+
+  private indexArr = [10, 14, 18];
+  private weekday = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```

Comment:
I'd suggest to move it to an enum. weekday.enum.ts or something like that

Reply:
done